mne_bids/write.py

```diff
@@ -1514,12 +1546,13 @@ def mark_bad_channels(ch_names, descriptions=None, *, bids_path,
     # Update info['bads']
     bads = _get_bads_from_tsv_data(tsv_data)
     raw.info['bads'] = bads
-    # XXX (How) will this handle split files?
     if isinstance(raw, mne.io.brainvision.brainvision.RawBrainVision):
-        events, event_id = _read_events(events_data=None, event_id=None,
-                                        raw=raw, verbose=False)
+        # XXX We should write durations too, this is supported by pybv.
+        events, _, _ = _read_events(events_data=None, event_id=None,
+                                    raw=raw, verbose=False)
         _write_raw_brainvision(raw, bids_path.fpath, events)
     elif isinstance(raw, mne.io.RawFIF):
+        # XXX (How) will this handle split files?
         raw.save(raw.filenames[0], overwrite=True, verbose=False)
```

Comment:
are you sure it does not work split files? With raw.load_data() it reads all split files and here you save it back.

Reply:
Never tried, honestly… I think we'd have to ensure we're using the same `split_size` as in the original. Is this parameter stored somewhere?

Reply:
@larsoner May know more :) Is there a reliable way post-hoc way to determine the `split_size` that was used when creating FIFF files?